geomat/stein/migrations/0057_remove_mineraltype_density.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.0.2 on 2018-04-25 07:44
+
+from django.db import migrations
+
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('stein', '0056_mineraltype_new_density'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='mineraltype',
+            name='density',
+        ),
+        migrations.RenameField(model_name="mineraltype", old_name="new_density", new_name="density")
```

Comment:
Der neue Name ist doch nun `new_density`? Zumindest ist das doch die Änderung weiter unten

Reply:
Nein.
1. neues Feld `new_density` erstellt.
2. Data Migration
3. altes Feld `density` gelöscht
4. `new_density` zu `density` umbenannt.

`old_name` entspricht dem Namen den das Feld momentan noch hat, und `new_name` ist der zu dem man es umbennen will. Also müsste das alles so stimmen.
Was allerdings noch Fehlt ist, die Änderung in `models.py`